code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -154,8 +162,9 @@
 			update_inv_wear_suit()
 		if(ITEM_SLOT_ICLOTHING)
 			if (w_uniform && swap)
-				w_uniform.dropped(src, TRUE)
 				current_equip = w_uniform
+				if (!temporarilyRemoveItemFromInventory(w_uniform))
+					return
```

Comment:
Very minor order thing, why do the check after assigning current_equip? It doesn't appear to be needed in the slightest, and is slightly confusing to read.

Reply:
:shrug: I can change it if you want.

Reply:
let it be known that i was right